src/intro.md

```diff
@@ -23,6 +23,10 @@ extensible. Moreover, these patterns provide a common language for developers,
 making them an excellent tool for effective communication when problem-solving
 in teams.
 
+Keep in mind: Each pattern comes with its own set of trade-offs.
+It's crucial to focus on why you choose a particular pattern rather than
+just on how to implement it.
+
```

Comment:
```suggestion
Keep in mind: Each pattern comes with its own set of trade-offs. It's crucial to
focus on why you choose a particular pattern rather than just on how to
implement it.

```